doc/install/install-ceph-gateway.rst

```diff
@@ -1,255 +1,583 @@
+	 "suspended": 0,
+	 "max_buckets": 1000,
+	 "auid": 0,
+	 "subusers": [{
+		 "id": "testuser:swift",
+		 "permissions": "full-control"
+	 }],
+	 "keys": [{
+		 "user": "testuser:swift",
+		 "access_key": "3Y1LNW4Q6X0Y53A52DET",
+		 "secret_key": ""
+	 }, {
+		 "user": "testuser",
+		 "access_key": "I0PJDPCIYZ665MW88W9R",
+		 "secret_key": "dxaXZ8U90SXydYzyS5ivamEP20hkLSUViiaR+ZDA"
+	 }],
+	 "swift_keys": [{
+		 "user": "testuser:swift",
+		 "secret_key": "244+fz2gSqoHwR3lYtSbIyomyPHf3i7rgSJrF\/IA"
+	 }],
+	 "caps": [],
+	 "op_mask": "read, write, delete",
+	 "default_placement": "",
+	 "placement_tags": [],
+	 "bucket_quota": {
+		 "enabled": false,
+		 "max_size_kb": -1,
+		 "max_objects": -1
+	 },
+	 "user_quota": {
+		 "enabled": false,
+		 "max_size_kb": -1,
+		 "max_objects": -1
+	 },
+	 "temp_url_keys": []
+ }
+
+Access Verification
+^^^^^^^^^^^^^^^^^^^
+
+Test S3 Access
+""""""""""""""
+
+You need to write and run a Python test script for verifying S3 access. The S3
+access test script will connect to the ``radosgw``, create a new bucket and
+list all buckets. The values for ``aws_access_key_id`` and
+``aws_secret_access_key`` are taken from the values of ``access_key`` and
+``secret_key`` returned by the ``radosgw_admin`` command.
+
+Execute the following steps:
+
+#. You will need to install the ``python-boto`` package::
+
+    sudo yum install python-boto
+
+#. Create the Python script::
+
+    vi s3test.py
+
+#. Add the following contents to the file::
+
+    import boto
+    import boto.s3.connection
+
+    access_key = 'I0PJDPCIYZ665MW88W9R'
+    secret_key = 'dxaXZ8U90SXydYzyS5ivamEP20hkLSUViiaR+ZDA'
+    conn = boto.connect_s3(
+            aws_access_key_id = access_key,
+            aws_secret_access_key = secret_key,
+            host = '{hostname}', port = {port},
+            is_secure=False, calling_format = boto.s3.connection.OrdinaryCallingFormat(),
+            )
+
+    bucket = conn.create_bucket('my-new-bucket')
+        for bucket in conn.get_all_buckets():
+                print "{name}".format(
+                        name = bucket.name,
+                        created = bucket.creation_date,
+     )
+
+   Replace ``{hostname}`` with the hostname of the host where you have
+   configured the gateway service i.e, the ``gateway host``. Replace {port}
+   with the port number you are using with Civetweb.
+
+#. Run the script::
+
+    python s3test.py
+
+   The output will be something like the following::
+
+    my-new-bucket 2015-02-16T17:09:10.000Z
+
+Test swift access
+"""""""""""""""""
+
+Swift access can be verified via the ``swift`` command line client. The command
+``man swift`` will provide more information on available command line options.
+
+To install ``swift`` client, execute the following commands. On Red Hat
+Enterprise Linux::
+
+ sudo yum install python-setuptools
+ sudo easy_install pip
+ sudo pip install --upgrade setuptools
+ sudo pip install --upgrade python-swiftclient
+
+On Ubuntu::
```

Comment:
nit, you could say Debian based distros, as this is used elsewhere, ok eitherway though.